karma.conf.ts

```diff
@@ -40,7 +40,13 @@ module.exports = (config) => {
 	config.set({
 		mode: "development",
 		browserNoActivityTimeout: 10000,
-		browsers: ["ChromeHeadless"],
+		browsers: ["ChromeDebugging"],
+		customLaunchers: {
```

Comment:
did this fix the chrome launching problem or was it playwright-chromium?

Reply:
Custom launcher "ChromeDebugging" is used only for debugging tests, someone in the future may use the configuration to debug tests too.

Changing dependency "playwright" to "playwright-chromium" actually fixed the errors in GitHub Actions.